packages/bsp/common/usr/lib/armbian/armbian-check-battery-shutdown

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -o nounset
+set -o errexit
+set -o pipefail
+shopt -s dotglob
+
+# SPDX-License-Identifier: MIT
+
+# This script should be executed every 5 minutes by a cron job `/etc/cron.d/armbian-check-battery`.
+# It checks if battery is discharging and battery level is more than 10%. If less, then start a system shutdown.
+# Script uses `batteryinfo` function from `30-armbian-sysinfo` file of Armbian distribution.
+
+BATTERY_PERCENT_MIN='10'
+
+# Import a script with a function `batteryinfo`:
+source /etc/update-motd.d/30-armbian-sysinfo
```

Comment:
`/etc/update-motd.d` is a bad place for utility function storage. `/usr/lib/armbian` looks much better.

Reply:
Guys, do you want me to move a part of `/etc/update-motd.d/30-armbian-sysinfo` into something like `/usr/lib/armbian/allwinner-battery.sh`?

Reply:
I've moved allwinner-related functions into `/usr/lib/armbian/armbian-allwinner-battery`.